runtime/src/accounts_db.rs

```diff
@@ -379,7 +384,6 @@ pub struct AccountsDB {
     /// the accounts
     min_num_stores: usize,
 
-    /// slot to BankHash and a status flag to indicate if the hash has been initialized or not
```

Comment:
A bit aggressive, but delete this comment as it's outdate and now adds no additional information by the comment.

- This comment is introduced at 19ae5568579135d69d3dfaf3a62209b8da271143 (#5573)
- This comment is forgot to be updated at 42da1ce4e23678cc683b54bd41935d8b58b88f09 (#7052)